src/app/ubs/ubs-admin/components/ubs-admin-table/table-cell-input/table-cell-input.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { IAlertInfo, IEditCell } from '@ubs/ubs-admin/models/edit-cell.model';
+import { IColumnBelonging } from '@ubs/ubs-admin/models/ubs-admin.interface';
+import { AdminTableService } from '@ubs/ubs-admin/services/admin-table.service';
+import { take } from 'rxjs';
+import { CommentPopUpComponent } from '../../shared/components/comment-pop-up/comment-pop-up.component';
+import { MatDialog, MatDialogConfig } from '@angular/material/dialog';
+import { LocalStorageService } from '@global-service/localstorage/local-storage.service';
+@Component({
+  selector: 'app-table-cell-input',
+  templateUrl: './table-cell-input.component.html',
+  styleUrls: ['./table-cell-input.component.scss']
+})
+export class TableCellInputComponent {
+  @Input() column: IColumnBelonging;
+  @Input() id: number;
+  @Input() ordersToChange: number[];
+  @Input() isAllChecked: boolean;
+  @Input() isUneditableStatus: boolean;
+  @Input() data: string;
+
+  @Output() cancelEdit = new EventEmitter();
+  @Output() editCommentCell = new EventEmitter();
+  @Output() showBlockedInfo = new EventEmitter();
+
+  isEditable: boolean;
+  isBlocked: boolean;
+
+  private typeOfChange: number[];
+  private font = '12px Lato, sans-serif';
```

Comment:
Member 'font' is never reassigned; mark it as `readonly`.
also pay attentions on sonar issues